pack/ramp-community.yml

```diff
@@ -6,7 +6,7 @@ description: High performance search index on top of redis
 homepage: http://redisearch.io
 license:  Redis Source Available License 2.0 (RSALv2) or the Server Side Public License v1 (SSPLv1) or the GNU Affero General Public License version 3 (AGPLv3)
 command_line_args: ""
-compatible_redis_version: "8.0"
+compatible_redis_version: "99.99.99"
```

Comment:
what about the version itself? (version.h)

Reply:
it is already 99.99.99 in master

Reply:
Are we going to keep it on master, or branch out?
So we will only run against latest redis, no more with 8.X?